### ISSUE TITLE
the app crash when using the selectPlaceList method

### ISSUE THREAD

**Issue by EmeryGuillaume** [Mapwize/mapwize-cordova#16]
@mathieugerard @laszloblum I have issues implementing the selectPlaceList method.
This behavior can be reproduced on the mapwize test app.
On iOS, open the place list: the app crash.
Here is the Xcode logs before the app crash:
```
2019-05-03 15:42:27.492861+0200 Mapwize Tester[442:45019] -[MWZPlaceList marker]: unrecognized selector sent to instance 0x282bff720
2019-05-03 15:42:27.493130+0200 Mapwize Tester[442:45019] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[MWZPlaceList marker]: unrecognized selector sent to instance 0x282bff720'
```

**Comment by mathieugerard** [comment]
Hi Guillaume. This is actually an issue in the native code. We are working on it.

**Comment by emerciercontexeo** [comment]
Fixed in MapwizeUI iOS 1.2.0.
